content/zh-cn/blog/_posts/2023-08-15-pkgs-k8s-io-introduction.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: blog
+title: "pkgs.k8s.io：介绍 Kubernetes 社区自有的包存储库"
+date: 2023-08-15T20:00:00+0000
+slug: pkgs-k8s-io-introduction
+---
+<!--
+layout: blog
+title: "pkgs.k8s.io: Introducing Kubernetes Community-Owned Package Repositories"
+date: 2023-08-15T20:00:00+0000
+slug: pkgs-k8s-io-introduction
+-->
+
+<!--
+**Author**: Marko Mudrinić (Kubermatic)
+-->
+**作者**：Marko Mudrinić (Kubermatic)
+
+**译者**：Wilson Wu (DaoCloud)
+
+<!--
+On behalf of Kubernetes SIG Release, I am very excited to introduce the Kubernetes community-owned software repositories for Debian and RPM packages: `pkgs.k8s.io`! The new package repositories are replacement for the Google-hosted package repositories (`apt.kubernetes.io` and `yum.kubernetes.io`) that we've been using since Kubernetes v1.5.
+-->
+我很高兴代表 Kubernetes SIG Release 介绍 Kubernetes
+社区自有的 Debian 和 RPM 软件包存储库：`pkgs.k8s.io`！
+新的包存储库取代了我们自 Kubernetes v1.5 以来一直使用的托管在
+Google 的包存储库（`apt.kubernetes.io` 和 `yum.kubernetes.io`）。
+
+<!--
+This blog post contains information about these new package repositories, what does it mean to you as an end user, and how to migrate to the new repositories.
+-->
+这篇博文包含关于这些新包存储库的信息、它对最终用户意味着什么以及如何迁移到新存储库。
+
+<!--
+**ℹ️  Update (August 31, 2023):** the _**legacy Google-hosted repositories are deprecated and will be frozen starting with September 13, 2023.**_ Check out [the deprecation announcement](/blog/2023/08/31/legacy-package-repository-deprecation/) for more details about this change.
+-->
+**ℹ️ 更新（2023 年 8 月 31 日）：旧版托管在 Google 的存储库已被弃用，并将于 2023 年 9 月 13 日开始被冻结。**
+查看[弃用公告](/zh-cn/blog/2023/08/31/legacy-package-repository-deprecation/)了解有关此更改的更多详细信息。
+
+<!--
+## What you need to know about the new package repositories?
+-->
+## 关于新的包存储库，你需要了解哪些信息？ {#what-you-need-to-know-about-the-new-package-repositories}
+
+<!--
+_(updated on August 31, 2023)_
+-->
+（更新于 2023 年 8 月 31 日）
```

Comment:
```suggestion
**（更新于 2023 年 8 月 31 日）**
```
The italic can be changed to Bold after localization.

Reply:
OK, add this will trigger a lint worning, but no problem for this repo.